examples/ex02_cksum/test/ChecksumRTL_test.py

```diff
@@ -12,6 +12,7 @@
 from pymtl3 import *
 from pymtl3.passes.tracing import VcdGenerationPass, PrintTextWavePass
 from pymtl3.stdlib.test_utils import TestSinkCL, TestSrcCL
+from pymtl3.stdlib.test_utils.test_helpers import _finalize
```

Comment:
Maybe not prefix `finalize` with an underscore?

Reply:
or change it to a different name ? finalize_import?

Reply:
it has been renamed to finalize_verilator